### PR TITLE
add unified keys

### PR DESCRIPTION
**Pull request by kkowalsky**
edit keys in _every demo_ to reflect new unified keys

### CODE REVIEW

**Comment by rmglennon** [comment]
LGTM. Thanks for sorting through this.

In the examples, the link text still shows a key (to give people an example/reminder to add it), but the actual link does not include a key because it relies on keyless access. This is a clean experience.

Note for future: with keyless access, does this http status code happen now? https://github.com/pelias/pelias-doc/blob/8f47034f4d1c25c4f58ad51e106f1f7fe9eb6e8a/api-keys-rate-limits.md#failing-to-provide-an-api_key

File: api-keys-rate-limits.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ To use the Mapzen Search service, you must first get a developer API key. Sign i
 
 1. Go to https://mapzen.com/developers.
 2. Sign in with your GitHub account. If you have not done this before, you need to agree to the terms first.
-3. Create a new key for Mapzen Search, and optionally, give it a name so you can remember the purpose of the project.
+3. Create a new key and optionally, give it a name so you can remember the purpose of the project.
```
Comment (trescube) [inline api-keys-rate-limits.md]: Either no comma or comma before "optionally" (I think).

Comment (rmglennon) [inline api-keys-rate-limits.md]: thanks. moved it around (which we now should do on all the instances...need smart content reuse variables!)

**Comment by dianashk** [comment]
LGTM (still needed for the approvals checks)

**Comment by rmglennon** [comment]
LGTM

**Comment by orangejulius** [comment]
LGTM

**Comment by trescube** [comment]
:shipit:

**Pull request merged**